src/features/controls.ts

```diff
@@ -22,8 +22,8 @@ import {ChangeEvent, SmoothControls} from '../three-components/SmoothControls.js
 import {Constructor} from '../utilities.js';
 
 export interface SphericalPosition {
-  theta: number;
-  phi: number;
+  theta: number;  // equator angle around the y (up) axis. Default is 0.
+  phi: number;    // polar angle from the y (up) axis. Default is 0.
```

Comment:
Good catch; I copied that out of three and didn't even notice the default. Mostly I put it in here because I was confused by the definition of phi. I'm used to elevation being zero at the equator, so we might want to put that in our public documentation as well.

Reply:
This was a carryover from `OrbitControls`. I would welcome a change to the defaults that seems more reasonable to do (although this would probably warrant a breaking change).

Reply:
Well, your defaults are actually 0 and 75 (15 degrees up from the equator), which seems pretty reasonable.